modules/photon-cdn/jetpack-manifest.php

```diff
@@ -1,505 +1,292 @@
-<?php 
-$assets = array (
-  0 => 'tools/jetpack-live-branches/jetpack-live-branches.user.js',
-  1 => 'tools/builder/frontend-css.js',
-  2 => 'tools/builder/transform-relative-paths.js',
-  3 => 'tools/builder/admin-css.js',
-  4 => 'bin/pre-commit-hook.js',
-  5 => 'css/jetpack-admin.min.css',
```

Comment:
I'm not sure what would build things in the CSS directory, but I think they just aren't on my local machine when this list is generated. I don't believe the directory is erroneously ignored.

Reply:
Oh, apparently I found them 🤷‍♂️